modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc

```diff
@@ -1,11 +1,12 @@
 // Module included in the following assembly:
 //
-// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+// jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc
 
-[id="jt-migrating-a-sample-pipeline-from-jenkins-to-tekton_{context}"]
-= Migrating a sample pipeline from Jenkins to Tekton
+:_content-type: PROCEDURE
+[id="jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines_{context}"]
+= Migrating a sample pipeline from Jenkins to OpenShift Pipelines
 
-This section provides equivalent examples of pipelines in Jenkins and Tekton and helps you to migrate your build, test, and deploy pipelines from Jenkins to Tekton.
+You can use the following equivalent examples to help migrate your build, test, and deploy pipelines from Jenkins to OpenShift Pipelines.
```

Comment:
is it relevant to use a `placeholder` for `OpenShift Pipelines` ? This way we can have a different documentation between enterprise and community.

Reply:
That's an interesting suggestion. I'll leave it to Souvik Sarkar (@sounix000), the primary writer for OpenShift Pipelines, whether to implement that change. If so, he'll do it in a separate doc task.